Khan/wonder-blocks
# Add stories for modals

There are no changes to the components themselves.  This PR only adds some storybook stories for modals so that we can some screenshots to avoid visual regressions.  It uses chromatic's `viewport` option to specify a three different browser widths to take screenshots at.  This PR also experiments with the viewport-addon for storybook which allows you to interactively select between different browser dimensions when running storybook locally which is more convenient then using the browsers built-in viewport emulation with styleguidist.

Test Plan:
- yarn start:storybook
- open one of the modal stories
- see that the viewport icon appears and that you can switch between viewports
- see that 3 snapshots were take per modal story for a total of 6 snapshots in Chromatic

**"simple" story on local storybook instance**
<img width="1542" alt="Screen Shot 2020-01-13 at 11 25 12 PM" src="https://user-images.githubusercontent.com/1044413/72314374-addc2b80-365c-11ea-9b44-4a97bc61bec4.png">

**chromatic snapshot of "kitchenSink" story"**
<img width="1401" alt="Screen Shot 2020-01-13 at 11 57 35 PM" src="https://user-images.githubusercontent.com/1044413/72315395-a4ed5900-3660-11ea-9d01-639651b4c3d8.png">
